Python/bytecodes.c

```diff
@@ -5251,7 +5251,8 @@ dummy_func(
                 printf("SIDE EXIT: [UOp ");
                 _PyUOpPrint(&next_uop[-1]);
                 printf(", exit %lu, temp %d, target %d -> %s]\n",
```

Comment:
For the pointers difference, you can use the `t` modifier:
```suggestion
                printf(", exit %tu, temp %d, target %d -> %s]\n",
```

Reply:
Thanks, just learn it!